src/pages/Settings/settings.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class settings extends React.Component {
+  render() {
+    return (
+      <div>
+        <h1 data-testid="settings-title">Configuração</h1>
```

Comment:
Penso que também seria interessante mantermos o mesmo texto do botão que leva à página.
```suggestion
        <h1 data-testid="settings-title">Configurações</h1>
```